scripts/001-prepare.sh

```diff
@@ -20,25 +20,24 @@ PROJECT_REPOSITORY=https://github.com/openstack/kolla
 PROJECT_REPOSITORY_PATH=kolla
 SOURCE_DOCKER_TAG=build-$BUILD_ID
 
-# NOTE: For builds for a specific release, the OpenStack version is taken from the release repository.
-if [[ $VERSION != "latest" ]]; then
-    filename=$(curl -L https://raw.githubusercontent.com/osism/release/main/$VERSION/openstack.yml)
-    OPENSTACK_VERSION=$(curl -L https://raw.githubusercontent.com/osism/release/main/$VERSION/$filename | grep "openstack_version:" | awk -F': ' '{ print $2 }')
-fi
-
 . defaults/all.sh
 . defaults/$OPENSTACK_VERSION.sh
 
 export VERSION
 export OPENSTACK_VERSION
 
-git submodule update --remote
+git clone https://github.com/osism/release
 
 if [[ ! -e release/$VERSION/base.yml ]]; then
     echo "release $VERSION does not exist"
     exit 1
 fi
 
+# NOTE: For builds for a specific release, the OpenStack version is taken from the release repository.
+if [[ $VERSION != "latest" ]]; then
+    OPENSTACK_VERSION=$(grep "openstack_version:" release/$VERSION/openstack.yml | awk -F': ' '{ print $2 }')
```

Comment:
Nit: If you use ': ' as field separator, you should also search for 'openstack_version: '.

Also the grep is redundant, you can add /openstack_version:/ to the awk statement instead.